### PR TITLE
chore: update the dependabot grouping

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -5,20 +5,51 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
-      time: "09:00"
-      timezone: "Europe/London"
-    commit-message:
-      prefix: "fix:"
-      prefix-development: "chore:"
```
Comment (alexmuller) [inline .github/dependabot.yml:12]: Is the removal of this right? I assumed these were here because of commitlint/release-please

Comment (rowanmanning) [reply .github/dependabot.yml:12]: Oops! That's very annoying of me

Comment (rowanmanning) [reply .github/dependabot.yml:12]: I'm going to add it to all of them. How irritating 😂

Comment (alexmuller) [reply .github/dependabot.yml:12]: Sorry!

**Review by alexmuller — APPROVED** [review]

**Pull request merged**